ui/src/organizations/components/GetOrgResources.tsx

```diff
@@ -97,6 +97,10 @@ export default class GetOrgResources<T> extends PureComponent<
     resources: T[],
     key: string | number | symbol
   ): key is keyof T {
-    return key in resources[0]
+    const resource = resources[0]
+    // gaurd against null and primitive types
+    const isObject = !!resource && typeof resource === 'object'
+
+    return isObject && key in resource
```

Comment:
an alternative could be to use lodash [_.keys](https://lodash.com/docs/4.17.11#keys) function to get the array of keys and check that `keys.includes(key)`

Reply:
Yeah I was going to add that but didn't like it's behavior with strings

```
_.keys("abc") // =>  ["0", "1", "3"]
```